roles/ec2-ami-base/Makefile

```diff
@@ -0,0 +1,85 @@
+SHELL := /usr/bin/env bash
```

Comment:
this file would belong in the 'deploy repo' for the ami if it is going to be used